webrtc/no-media-call.html

```diff
@@ -0,0 +1,132 @@
+<!doctype html>
+<!--
+This test uses no media, and thus does not require fake media devices.
+-->
+
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>RTCPeerConnection No-Media Connection Test</title>
+</head>
+<body>
+  <div id="log"></div>
+  <div>
+    <video id="local-view" autoplay="autoplay"></video>
+    <video id="remote-view" autoplay="autoplay"/>
+    </video>
+  </div>
+
+  <!-- These files are in place when executing on W3C. -->
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/common/vendor-prefix.js"
+          data-prefixed-objects=
+              '[{"ancestors":["navigator"], "name":"getUserMedia"},
+                {"ancestors":["window"], "name":"RTCPeerConnection"},
+                {"ancestors":["window"], "name":"RTCSessionDescription"},
+                {"ancestors":["window"], "name":"RTCIceCandidate"}]'
+          data-prefixed-prototypes=
+               '[{"ancestors":["HTMLMediaElement"],"name":"srcObject"}]'>
+  </script>
+  <script type="text/javascript">
+  var test = async_test('Can set up a basic WebRTC call with no data.',
+                        {timeout: 5000});
```

Comment:
why the 5000ms timeout? it's worth documenting the reason with a comment I think